package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-overlays",
-  "version": "1.0.5",
+  "version": "1.1.0",
```

Comment:
@ef4 i bumped this, but maybe you want to rev it when you publish the package?

Reply:
Yeah, in general it's presumptuous to do this in a feature PR. 😛It's not a big deal, I can adjust when merging.

Reply:
@ef4 i undid this change.